modules/platforms/cpp/cmake/dependencies.cmake

```diff
@@ -20,7 +20,7 @@ set(MBEDTLS_AS_SUBPROJECT ON)
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-add_compile_definitions(MBEDTLS_USER_CONFIG_FILE="${CMAKE_CURRENT_SOURCE_DIR}/ignite/common/ignite_mbedtls_config.h")
+add_compile_definitions(MBEDTLS_USER_CONFIG_FILE="${CMAKE_CURRENT_SOURCE_DIR}/ignite/common/detail/ignite_mbedtls_config.h")
```

Comment:
Here and below: unrelated changes? Do we need this for Doxyfile?

Reply:
Well, it's not directly related, but there were files in public space which should not be public, and Doxygen would generate documentation using them. I could exclude them manually, but the right way looked like moving them to `detail` folder (ie `impl`)